### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by halucinka**
Ahoj,

implementovala som funkciu na parsovanie bank statement emailov. Napisala som aj unit testy (s Davidkom:)).

Snazila som sa to spravit najrobustnejsie ako som to vedela. Je to ok?
Mam par otazok:
(1) Treba brat do uvahy aj stary format cisel uctu? Spravila som zatial iba IBAN.
(2) Netusim, co je transaction description - je to sparsovane ako string bez akychkolvek uprav.
(3) Nie je mi jasne aky vystupny objekt si si predstavoval. Napisala som class-u.

L.

### CODE REVIEW

File: processing_notifications.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+# Lucia Pal, 2015
+
+'''
+Module for parsing Tatra Banka bank statements emails.
+'''
+
+import re
+
+class Transaction():
```
Comment (hleumas) [inline processing_notifications.py]: I would just use dictionary. Don't see much value in `class` here.

File: processing_notifications.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+	Converts an email message to a transaction.
+
+	We ignore white spaces (including end of lines and tabs) and the '>' character except in the message fields themselves.
+	'''
+
+	def parse_account_number(email):
+		'''
+		Finds the account number in the email message.
+
+		Current implementation works only for IBAN account numbers. See: https://en.wikipedia.org/wiki/International_Bank_Account_Number
```
Comment (hleumas) [inline processing_notifications.py]: Try to stick to lines <80 characters long. Helps reading them:)

**Comment by hleumas** [comment]
Thank you for nice code. Please, squash commits together, write nice meaningful commit message and feel free to merge into master.

http://chris.beams.io/posts/git-commit/

**Comment by halucinka** [comment]
I've replaced transaction class with a dictionary. I've reformatted to <80 characters per line. I've squashed the commits into a single one.

**Comment by halucinka** [comment]
Merging into master.

**Pull request merged**